dbinit.sql

```diff
@@ -4,130 +4,97 @@ SET sql_safe_updates = false;
-    CONSTRAINT fk_city_ref_users FOREIGN KEY (rider_city, rider_id) REFERENCES users(city, id),
-    CONSTRAINT fk_vehicle_city_ref_vehicles FOREIGN KEY (city, vehicle_id) REFERENCES vehicles(city, id),
-    INDEX rides_auto_index_fk_city_ref_users (rider_city ASC, rider_id ASC) PARTITION BY LIST (rider_city) (
-        PARTITION us_west VALUES IN (('seattle'), ('san francisco'), ('los angeles')),
-        PARTITION us_east VALUES IN (('new york'), ('boston'), ('washington dc')),
-        PARTITION europe_west VALUES IN (('amsterdam'), ('paris'), ('rome'))
-    ),
-    INDEX rides_auto_index_fk_vehicle_city_ref_vehicles (city ASC, vehicle_id ASC) PARTITION BY LIST (city) (
-        PARTITION us_west VALUES IN (('seattle'), ('san francisco'), ('los angeles')),
-        PARTITION us_east VALUES IN (('new york'), ('boston'), ('washington dc')),
-        PARTITION europe_west VALUES IN (('amsterdam'), ('paris'), ('rome'))
-    ),
-    FAMILY "primary" (id, city, rider_id, rider_city, vehicle_id, start_location, end_location, start_time, end_time, length)
-)  PARTITION BY LIST (city) (
-        PARTITION us_west VALUES IN (('seattle'), ('san francisco'), ('los angeles')),
-        PARTITION us_east VALUES IN (('new york'), ('boston'), ('washington dc')),
-        PARTITION europe_west VALUES IN (('amsterdam'), ('paris'), ('rome'))
-);
-ALTER PARTITION europe_west OF INDEX movr.public.rides@primary CONFIGURE ZONE USING
-    constraints = '[+region=gcp-europe-west1]';
-ALTER PARTITION us_east OF INDEX movr.public.rides@primary CONFIGURE ZONE USING
-    constraints = '[+region=gcp-us-east1]';
-ALTER PARTITION us_west OF INDEX movr.public.rides@primary CONFIGURE ZONE USING
-    constraints = '[+region=gcp-us-west1]';
-ALTER PARTITION europe_west OF INDEX movr.public.rides@rides_auto_index_fk_city_ref_users CONFIGURE ZONE USING
-    constraints = '[+region=gcp-europe-west1]';
-ALTER PARTITION us_east OF INDEX movr.public.rides@rides_auto_index_fk_city_ref_users CONFIGURE ZONE USING
-    constraints = '[+region=gcp-us-east1]';
-ALTER PARTITION us_west OF INDEX movr.public.rides@rides_auto_index_fk_city_ref_users CONFIGURE ZONE USING
-    constraints = '[+region=gcp-us-west1]';
-ALTER PARTITION europe_west OF INDEX movr.public.rides@rides_auto_index_fk_vehicle_city_ref_vehicles CONFIGURE ZONE USING
-    constraints = '[+region=gcp-europe-west1]';
-ALTER PARTITION us_east OF INDEX movr.public.rides@rides_auto_index_fk_vehicle_city_ref_vehicles CONFIGURE ZONE USING
-    constraints = '[+region=gcp-us-east1]';
-ALTER PARTITION us_west OF INDEX movr.public.rides@rides_auto_index_fk_vehicle_city_ref_vehicles CONFIGURE ZONE USING
-    constraints = '[+region=gcp-us-west1]'
-;
+    CONSTRAINT fk_city_ref_users FOREIGN KEY (rider_id) REFERENCES users(id),
+    CONSTRAINT fk_vehicle_ref_vehicles FOREIGN KEY (vehicle_id) REFERENCES vehicles(id)
+    );
```

Comment:
nit: unindent ending `);`

Reply:
After fixing the table (see [comment below](https://github.com/cockroachlabs/movr-flask/pull/27#discussion_r621261601)), I formatted the document with [pgformatter](https://github.com/darold/pgFormatter). Should look nicer all over now.